src/main/java/Filters/DashboardFilter.java

```diff
@@ -6,242 +6,50 @@
 import javax.servlet.http.HttpServletResponse;
 import javax.servlet.http.HttpSession;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.sql.SQLException;
-import java.util.List;
+import java.util.Arrays;
 
-import Models.Approval.Approval;
-import Models.Approval.ApprovalDAO;
-import Models.Event.Event;
-import Models.Event.EventDao;
-import Models.Event.Log;
-import Models.Prescription.Prescription;
-import Models.Prescription.PrescriptionDAO;
-import Models.Schedule.Schedule;
-import Models.Schedule.ScheduleDAO;
+import Controllers.DashboardDispatcher.AdminDispatcher;
+import Controllers.DashboardDispatcher.PatientDispatcher;
+import Controllers.DashboardDispatcher.StaffDispatcher;
 import Models.User.*;
-import static Utils.Passwords.createSaltAndHash;
-import Utils.Tables;
-import Utils.Tuple;
-import java.security.NoSuchAlgorithmException;
-import java.security.spec.InvalidKeySpecException;
-import java.util.Arrays;
-import java.util.Locale;
-import java.util.logging.Level;
-import java.util.logging.Logger;
 
 @WebFilter(filterName = "DashboardFilter")
 public class DashboardFilter implements Filter {
 
+    private final String[] pages = {"appointments", "patientinformation"};
+
     @Override
-    public void destroy() {
-    }
+    public void destroy() {}
```

Comment:
You and my autoformat appear to be battling it out here lmao

Reply:
You can change that in the Intellij settings - both are okay in the java formatting standards I think. I prefer single line empty to multiline

Reply:
But I'm just saying mine's gonna win